config/locales/eng.yml

```diff
@@ -109,6 +109,12 @@ eng:
         dear: 'Dear'
         thank-you: 'Your Sara Alert monitoring is now complete! Thank you for your participation.'
         footer: 'Do not reply to this email, forward this email, or share this link. This message was automatically generated by the Sara Alert system and is unique and intended only for you. If you wish to stop receiving these notifications or believe that it was a mistake, please contact your local public health department. For privacy and more info, visit saraalert.org.'
+      contact:
+        info1: 'If you have any questions, please use the contact information below to reach out to the public health organization that helped enroll you:'
+        info2: 'If you have any questions please reach out to the public health organization that helped enroll you.'
+        email: 'Email: '
```

Comment:
Can we move the `: ` into the html?

Reply:
changed in https://github.com/SaraAlert/SaraAlert/pull/925/commits/8b13a733d24e2e5aef59ae932e5bc50f05892c94